### PR TITLE
Bumps TF version to 0.7.13 and adds versioning

### PR DESCRIPTION
**Pull request by stevenjack**
### Problem

We want to use version `0.7.13` of terraform and bring the repo inline with our versioning workflow.

### Solution

* Bump TF version
* Add versioning

### CODE REVIEW

**Comment by stevenjack** [comment]
[[AR 1.6.0] Bumps TF and adds versioning](https://trello.com/c/w13Moli3/963-ar-1-6-0-bumps-tf-and-adds-versioning)

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+BRANCH ?= "master"
+VERSION ?= $(shell cat ./VERSION)
+REPONAME ?= "assume-role"
+
+DEFAULT: run
+
+build:
```
Comment (revett) [inline Makefile]: Change to `build-image`

Comment (revett) [inline Makefile]: Also add a `build-image-local` task

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	@docker build -t vidsyhq/${REPONAME} .
+
+deploy:
+	@docker login -e ${DOCKER_EMAIL} -u ${DOCKER_USER} -p ${DOCKER_PASS}
+	@docker tag vidsyhq/${REPONAME}:latest vidsyhq/${REPONAME}:${CIRCLE_TAG}
+	@docker push vidsyhq/${REPONAME}:${CIRCLE_TAG}
+	@docker push vidsyhq/${REPONAME}
+
+check-version:
```
Comment (revett) [inline Makefile]: 🏄 order tasks alphabetically

**Pull request merged**